addons/viewdistance/initSettings.sqf

```diff
@@ -0,0 +1,73 @@
+private _distArr = [
+    [0,                      1,     2,      3,      4,      5,      6,      7,      8,      9,      10,     11,     12,     13,     14],
+    [LSTRING(videosettings), "500", "1000", "1500", "2000", "2500", "3000", "3500", "4000", "5000", "6000", "7000", "8000", "9000", "10000"],
+    0
```

Comment:
There are no official guidelines on array formating IIRC but we don't do this in the codebase usually. Can you format the array "normally".